packages/api/engine/opensea.go

```diff
@@ -72,38 +72,74 @@ func (o *Opensea) Sync(ctx context.Context) {
 
 				for _, asset := range ret.Assets {
 					var amount *big.Int
+					onSale := false
 					if asset.SellOrders != nil {
 						amount = asset.SellOrders[0].CurrentPrice.Big()
 						fmt.Println(asset.SellOrders[0].CurrentPrice)
+						onSale = true
 					} else {
 						amount = big.NewInt(0)
 					}
```

Comment:
```suggestion
					amount := big.NewInt(0)
					onSale := false
					if asset.SellOrders != nil {
						amount = asset.SellOrders[0].CurrentPrice.Big()
						fmt.Println(asset.SellOrders[0].CurrentPrice)
						onSale = true
					}
```

minimize branching when possible